notebooks/learning_graph_structured_data_h2.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>(Optional) Maybe give a brief description (one sentence) of what is meant by embedding a graph: embedding its nodes, so that...? Or a link to a (your?) paper?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I agree; we should make it really clear at the beginning. Do you feel like the new introduction better orients the reader to these ideas and what's to come?